apps/storybook/src/TiledHeatmap.stories.tsx

```diff
@@ -116,23 +116,29 @@ class MandelbrotTilesApi extends TilesApi {
     return this.store.get({ layer, offset });
   }
 }
-const Template: Story<TiledHeatmapProps> = (args) => {
-  const { api, ...tiledHeatmapProps } = args;
-  const { width, height } = api.baseLayerSize;
+
+interface TiledHeatmapStoryProps extends TiledHeatmapProps {
+  abscissaDomain: Domain;
+  ordinateDomain: Domain;
+}
+
+const Template: Story<TiledHeatmapStoryProps> = (args) => {
+  const { abscissaDomain, api, ordinateDomain, ...tiledHeatmapProps } = args;
 
   return (
     <VisCanvas
       abscissaConfig={{
-        visDomain: [0, width],
-        isIndexAxis: true,
+        visDomain: abscissaDomain,
         showGrid: false,
       }}
       ordinateConfig={{
-        visDomain: [0, height],
-        isIndexAxis: true,
+        visDomain: ordinateDomain,
         showGrid: false,
       }}
-      visRatio={width / height}
+      visRatio={Math.abs(
+        (abscissaDomain[1] - abscissaDomain[0]) /
+          (ordinateDomain[1] - ordinateDomain[0])
+      )}
```

Comment:
The ratio you want to keep should not be `api.baseLayerSize.width / api.baseLayerSize.height` ?

In any case, I think the story would allow to catch more regressions if the span of the abscissa domain was different from the span of the ordinate domain (so far, both span 3).

Reply:
> The ratio you want to keep should not be api.baseLayerSize.width / api.baseLayerSize.height ?

I expect most of the time, but maybe not... But it behaves poorly if there is a large difference between the axes ratio and the ratio of the image since the level of details to then differs between the axes.

> I think the story would allow to catch more regressions if the span of the abscissa domain was different from the span of the ordinate domain

:+1: